.github/workflows/main.yml

```diff
@@ -31,8 +31,18 @@ jobs:
           WF_RESOURCES=${{ inputs.workflow_id }}/
           QUARKUS_EXTENSIONS=org.kie.kogito:kogito-addons-quarkus-jobs-knative-eventing:999-SNAPSHOT
 
+    - name: Buildah push to OCI Arcive
+      if: ${{ ! inputs.push_pr }}
+      run: buildah push oci-archive:serverless-workflow-${{ inputs.workflow_id }}.tar:${{ github.sha }}
```

Comment:
how would the output of this be used as an image to be deployed on a running env?